### PR TITLE
Release the Python REST Client for REST 1.12

### PR DESCRIPTION
**Pull request by asun-ps**
tested manually and seems to work, also looked at documentation and things seem to show up correctly

### CODE REVIEW

File: PKG-INFO [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Metadata-Version: 1.11.0
+Metadata-Version: 1.12
```
Comment (ychen-ps) [inline PKG-INFO]: This needs to be manually updated to 1.12.0.

File: PKG-INFO [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 Name: purity_fb
-Version: 1.11.0
-Summary: Pure Storage FlashBlade REST 1.11 Python SDK.
+Version: 1.12
```
Comment (ychen-ps) [inline PKG-INFO]: Same with this one.

File: PKG-INFO [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
+Summary: A python client for Purity//FB REST API.
```
Comment (ychen-ps) [inline PKG-INFO]: Update the summary here to `Pure Storage FlashBlade REST 1.12 Python SDK.`

File: purity_fb/purity_fb_1dot5/models/policy_patch.py [hunk 4]
```diff
@@ -36,88 +36,32 @@ class PolicyPatch(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        'name': 'str',
```
Comment (tlewis-ps) [inline purity_fb/purity_fb_1dot5/models/policy_patch.py:39]: We should add to the release notes that we removed the unsupported `name` and `id` properties from PolicyPatch.

File: purity_fb/purity_fb_1dot11/apis/hardware_api.py [hunk 5]
```diff
@@ -3,7 +3,7 @@
 """
     Pure Storage FlashBlade REST 1.11 Python SDK
 
-    Pure Storage FlashBlade REST 1.11 Python SDK, developed by [Pure Storage, Inc](http://www.purestorage.com/). Documentations can be found at [purity-fb.readthedocs.io](http://purity-fb.readthedocs.io/).
```
Comment (tlewis-ps) [inline purity_fb/purity_fb_1dot11/apis/hardware_api.py:6]: Ah...This is why this PR is roughly twice as large as it should be...We must've missed updating the description for 1.11.

File: purity_fb/README.md [hunk 6]
```diff
@@ -606,10 +650,7 @@ Class | Method | HTTP request | Description
  - [PolicyMemberResponse](docs/PolicyMemberResponse.md)
  - [PolicyMemberWithRemote](docs/PolicyMemberWithRemote.md)
  - [PolicyMemberWithRemoteResponse](docs/PolicyMemberWithRemoteResponse.md)
- - [PolicyObjects](docs/PolicyObjects.md)
```
Comment (tlewis-ps) [inline purity_fb/README.md]: I _think_ that these files are actually still around and should be included. The code generate just drops them for some reason.
Can you confirm that these .md files exist, and then manually re-add them if they do?

File: docs/index.md [hunk 7]
```diff
@@ -606,17 +649,13 @@ Class | Method | HTTP request | Description
  - [PolicyMemberResponse](PolicyMemberResponse.md)
  - [PolicyMemberWithRemote](PolicyMemberWithRemote.md)
  - [PolicyMemberWithRemoteResponse](PolicyMemberWithRemoteResponse.md)
- - [PolicyObjects](PolicyObjects.md)
```
Comment (tlewis-ps) [inline docs/index.md]: Same as in `README.md`. These files should probably be manually re-added.

**Review by tlewis-ps — CHANGES_REQUESTED** [review]

File: purity_fb/purity_fb_1dot5/models/policy_patch.py (hunk 4 above)
Comment (asun-ps) [reply purity_fb/purity_fb_1dot5/models/policy_patch.py:39]: added to docs/index.md, README.md and purity_fb/README.md

File: purity_fb/README.md (hunk 6 above)
Comment (asun-ps) [reply purity_fb/README.md]: yup they exist, manually re-added

**Review by tlewis-ps — APPROVED** [review]
Looks good to me!

**Pull request merged**